app/[locale]/enterprise/page.tsx

```diff
@@ -0,0 +1,485 @@
+import WFP from "@/components/icons/enterprise/wfp.svg"
+import MainArticle from "@/components/MainArticle"
+import Translation from "@/components/Translation"
+import { ButtonLink } from "@/components/ui/buttons/Button"
+import { Card } from "@/components/ui/card"
+import { Skeleton, SkeletonLines } from "@/components/ui/skeleton"
+
+import { cn } from "@/lib/utils/cn"
+// import { dataLoader } from "@/lib/utils/data/dataLoader"
+import { getMetadata } from "@/lib/utils/metadata"
+
+// import { BASE_TIME_UNIT } from "@/lib/constants"
+import CasesColumn from "./_components/CasesColumn"
+import FeatureCard from "./_components/FeatureCard"
+import { ENTERPRISE_MAILTO } from "./constants"
+import type { Case, EcosystemPlayer, Feature } from "./types"
+import { parseActivity } from "./utils"
+
+import { fetchEthereumStablecoinsMcap } from "@/lib/api/fetchEthereumStablecoinsMcap"
+import { fetchEthPrice } from "@/lib/api/fetchEthPrice"
+import { fetchGrowThePie } from "@/lib/api/fetchGrowThePie"
+import { fetchTotalEthStaked } from "@/lib/api/fetchTotalEthStaked"
+import EthGlyph from "@/public/images/assets/svgs/eth-diamond-rainbow.svg"
+import heroImage from "@/public/images/heroes/enterprise-hero-white.png"
+
+const FeaturesSwiper = dynamic(() => import("./_components/FeaturesSwiper"), {
+  ssr: false,
+  loading: () => (
+    <Card className="ms-4 w-[85vw] rounded-4xl border bg-background px-2 py-6">
+      <Skeleton className="mx-4 mb-6 size-16 rounded-xl" />
+      <Skeleton className="mx-4 -mb-2 h-5 w-[85%]" />
+      <SkeletonLines noOfLines={7} />
+    </Card>
+  ),
+})
+
+const CasesSwiper = dynamic(() => import("./_components/CasesSwiper"), {
+  ssr: false,
+  loading: () => (
+    <Card className="ms-4 w-[85vw] space-y-1 rounded-4xl border bg-background px-2 py-6">
+      <Skeleton className="mx-4 h-5 w-[85%]" />
+      <SkeletonLines noOfLines={4} />
+    </Card>
+  ),
+})
+
+// TODO: Switch back to this for cache and mock-data dev fallback
+// const loadData = dataLoader(
+//   [
+//     ["growThePieData", fetchGrowThePie],
+//     ["ethereumStablecoins", fetchEthereumStablecoinsMcap],
+//     ["ethPrice", fetchEthPrice],
+//     ["totalEthStaked", fetchTotalEthStaked],
+//   ],
+//   BASE_TIME_UNIT * 1000
+// )
+
+const Page = async ({ params }: { params: { locale: Lang } }) => {
+  const { locale } = params
+
+  const t = await getTranslations({ locale, namespace: "page-enterprise" })
+
+  // const [{ txCount }, stablecoinMarketCap, ethPrice, totalEthStaked] =
+  //   await loadData()
+  const { txCount } = await fetchGrowThePie()
+  const stablecoinMarketCap = await fetchEthereumStablecoinsMcap()
+  const ethPrice = await fetchEthPrice()
+  const totalEthStaked = await fetchTotalEthStaked()
+
+  const metrics = await parseActivity(
+    {
+      txCount,
+      stablecoinMarketCap,
+      ethPrice,
+      totalEthStaked,
+      // totalCapitalSecured,
+    },
+    locale
+  )
```

Comment:
Just checking if this TODO needs to be handled before merged

Reply:
No, this was boilerplate for a "Total Value Secured" endpoint but we don't have one available just yet, so staked ETH is taking it's place for now.